File: katt/mastermind/submissions/accepted/mastermind.py
#!/usr/bin/env python2
import grader

def init(n):
    g = [False] * n
    cor = grader.guess(g)
    for i in range(n):
        g[i] = 1
        cor2 = grader.guess(g)
        if cor2 < cor: g[i] = 0
        else: cor = cor2

File: katt/mastermind/submissions/accepted/mastermind.java
class mastermind {
	public static void init(int N) {
		boolean[] g = new boolean[N];
		int cor = grader.lib.guess(g);
		for (int i = 0; i < N; i++) {
			g[i] = true;
			int cor2 = grader.lib.guess(g);
			if (cor2 < cor) g[i] = false;
			else cor = cor2;
		}
	}
}
